### ISSUE TITLE
SVG's hardcoded title is "leaking" for Icon

### ISSUE THREAD

**Issue by tomaszlukasik** [brainly/style-guide#2316]
It might be a duplicate of #1651 but there are not many details there, so it's hard to say.

I noticed that the tooltip shown when the mouse pointer is placed over the Icon element does not always display the correct title.

**How to reproduce:**
Just use any `Icon` and set a custom title: `<Icon title="Not a heart" type="heart" color="icon-black" size={40} />`

Placing the pointer at the edge of `<svg>` element will display a tooltip saying `Not a heart`, but placing the pointer over the `<use>` element will show a `heart` tooltip.

The svgs that are used, have [title](https://github.com/brainly/style-guide/blob/master/src/images/icons/heart.svg?short_path=ea04438#L2) "hardcoded", so when they are "embeded" by [use](https://github.com/brainly/style-guide/blob/master/src/components/icons/Icon.jsx#L606), those titles "leak".

**Examples** from tutoring:
![image](https://user-images.githubusercontent.com/1143912/155318365-d364f515-9dbd-49e2-8734-844bd5dc5a0a.png)
![image](https://user-images.githubusercontent.com/1143912/155318410-d2bce64c-5936-47dd-9444-8e04482fe0ae.png)

![image](https://user-images.githubusercontent.com/1143912/155318496-6ca8cc69-81a5-4354-940b-28f466bb7b4d.png)
![image](https://user-images.githubusercontent.com/1143912/155318530-b5038b42-88e4-49fd-8003-28a31f485feb.png)
![image](https://user-images.githubusercontent.com/1143912/155318586-03575980-5272-412f-8aac-c7f425624a5e.png)
https://github.com/brainly/style-guide/blob/master/src/images/icons/drawing_mode.svg?short_path=7bd5689#L2
